src/core/comm/coll.cc

```diff
@@ -52,7 +52,7 @@ static int mpi_tag_ub = 0;
 
 static std::vector<MPI_Comm> mpi_comms;
 #else  // undef LEGATE_USE_GASNET
-static std::vector<ThreadComm> thread_comms;
+static std::vector<ThreadComm*> thread_comms;
```

Comment:
does this vector ever get relocated? Did you try running this multi-node?

Reply:
Line 54 shows this vector is used for single node run only.

Reply:
I see, Ok